cishouseholds/pipeline/survey_responses_version_2_ETL.py

```diff
@@ -999,6 +1007,9 @@ def fill_forwards_transformations(df):
         "work_health_care_v1_v2",
         "work_nursing_or_residential_care_home",
         "work_direct_contact_patients_clients",
+        "sex",
+        "date_of_birth",
+        "ethnicity",
     ]
     df = fill_forward_only_to_nulls_in_dataset(
```

Comment:
I've just noticed that this filling method is based on `work_main_job_changed`. The fill forward of these variables should occur regardless of voyager version and independently to other columns.

Please add a new function call for their fill forwards after the `fill_backwards_overriding_not_nulls` call below. Note that this call is currently duplicated above and below